scripts/sample-file-for-getScreedId-usage.js

```diff
@@ -0,0 +1,187 @@
+   // via: https://bugs.chromium.org/p/chromium/issues/detail?id=487935#c17
+                // you can capture screen on Android Chrome >= 55 with flag: "Experimental ScreenCapture android"
+                window.IsAndroidChrome = false;
+                try {
+                    if (navigator.userAgent.toLowerCase().indexOf("android") > -1 && /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor)) {
+                        window.IsAndroidChrome = true;
+                    }
+                } catch (e) {}
+
+                document.getElementById('capture-screen').onclick = function() {
+                    document.getElementById('capture-screen').disabled = true;
+
+                    setTimeout(function() {
+						
+                      if(document.getElementById('capture-screen').disabled && !document.querySelector('video').src) {
+                        document.getElementById('capture-screen').disabled = false;
+                      }
+					  
+                    }, 5000);
+			
+			var tts= new SpeechSynthesisUtterance("OK GAurav");
+			var BoundingFrequencyMale = [0,500,700,3000,6000];//[0,400,560,2400,4800];
+			var BoundingFrequencyFemale = [0,500,700,3000,6000]; 
+			var BlendShapes ={};
+
+
+			var audioContext = new AudioContext();
+			var FFT_SIZE = 1024;
+			var samplingFrequency = 44100;
+			var Error= function(){console.log("failed to get Media");}
+
+			var IndicesFrequencyFemale = [];
+			var IndicesFrequencyMale = [];
+			for( let m = 0 ; m < BoundingFrequencyMale.length ; m++) {
+			   IndicesFrequencyMale[m] = Math.round(((2*FFT_SIZE)/samplingFrequency) *BoundingFrequencyMale[m]);
+			   //IndicesFrequencyFemale[n] = ((2*FFT_SIZE)/samplingFrequency) *BoundingFrequencyFemale[m];
+			   console.log("IndicesFrequencyMale[",m,"]",IndicesFrequencyMale[m]);
+			}
+			getRMS = function (spectrum) {
+			var rms = 0;
+			  for (var i = 0; i < spectrum.length; i++) {
+			    rms += spectrum[i] * spectrum[i];
+			  }
+			  rms /= spectrum.length;
+			  rms = Math.sqrt(rms);
+			  return rms;
+ 			}
+
+			getstPSD = function (spectrum) {
+			  var sensitivity_threshold = 0.5;
+			   var    stPSD = new Float32Array(spectrum.length);
+				for ( i = 0 ; i< spectrum.length;i++) {
+				   stPSD[i]= sensitivity_threshold + ((spectrum[i] +20)/140);	
+				}
+			  return stPSD;
+ 			}
+                    getScreenId(function(error, sourceId, screen_constraints) {
+                        // error    == null || 'permission-denied' || 'not-installed' || 'installed-disabled' || 'not-chrome'
+                        // sourceId == null || 'string' || 'firefox'
+                        // getUserMedia(screen_constraints, onSuccess, onFailure);
+
+                        document.getElementById('capture-screen').disabled = false;
+
+                        if (IsAndroidChrome) {
```

Comment:
Which platforms' builds of chrome support extensions (and screen sharing)? Do we need to do different things for windows, mac, linux, android and iOS?

Reply:
Windows,mac linux has no problem "only" for Android chrome i made exception. 

Also this is just a sample file to explain usage of getScreenId() in application where we are receiving the screenid from chrome extension  and passing it in getUserMedia API as constraints. This should be replaced in HUBs code where we are directly calling getUserMedia for screensharing.